src/python/rai/rai/dicom/uid.py

```diff
@@ -32,9 +32,13 @@
 _RAI_CONTOURS_PRODUCT_ID = "1"
 
 RAI_CLIENT_ROOT_UID_PREFIX = f"{_RAI_ROOT_UID_PREFIX}1."
-RAI_IMPLEMENTATION_VERSION_NAME = f"rai-v{__version__}"
+
+# TODO: Add a representation within the implementation uid of production
+# vs dev release.
+_NO_DEV_VERSION = __version__.split("-", maxsplit=1)[0]
```

Comment:
Does this mean your re going to have some UID that ends with .0 while still in pre-release?
That's OK, but be careful about ending up with leading zeros or double zeros

Reply:
Yup, will do.

This still does have that assert in there, so it should scream at me now if the UID isn't valid.